mdflynn/rancid-tomatillos
# Feature/display video

## Is this a fix or a feature?
- Feature, fix
## What is the change?
- The application has been changed to make user of `react-router`
- The test files have been updated to reflect the switch over to 'react-router`
- Movie Trailers are now properly rendered to the `<Movie/>`

## What does it fix?
- #5, #19, #18, #21

## Where should the reviewer start?
- App.js: Lines 16-133
- App.test.js: Lines 1-141
- Form.test: Lines 1-14
- Movie.js: Lines 1-29
- Movie.js: Lines 1-23
- Movie.test.js: Lines 1-59
- Nav.js: Lines 1-20
- Nav.test.js: Lines 1-34
- callAPI: Lines 12-21
## How should this be tested?
- Clone down the repo
- `cd` into the directory
- Run `npm start` 
- Navigate to the provided localhost url 
- Type the name of a movie into the search field and observe how the movies are filtered based on your entry
- Click on a movie
- Notice the trailer that appears on the right side of the page
- Play the trailer if you desire
